programs/tic-tac-toe/programs/tic-tac-toe/src/lib.rs

```diff
@@ -87,7 +87,7 @@ impl Game {
 
         self.update_state();
 
-        if let GameState::Active = self.state {
+        if GameState::Active = self.state {
```

Comment:
```suggestion
        if GameState::Active == self.state {
```
Should be `==`? :thinking:

Reply:
yeah